etc/spack/defaults/darwin/packages.yaml

```diff
@@ -21,11 +21,14 @@ packages:
     - gcc
     - intel
     providers:
-      elf: [libelf]
-      unwind: [apple-libunwind]
+      elf:
+      - libelf
+      unwind:
+      - apple-libunwind
   apple-libunwind:
-    paths:
+    buildable: false
```

Comment:
Are false and False equivalent?

Reply:
Yes, per YAML v1.2 specifications `true` and `false` should be the canonical values used by YAML, but anything matching the regexp:
```
true | True | TRUE | false | False | FALSE
```
is treated as a boolean. Standard reference [here](https://yaml.org/spec/1.2/spec.html#id2803629) and a nice [SO answer](https://stackoverflow.com/a/44213322/771663) I found while looking for the previous reference.